.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,52 @@
+---
+name: Bug Report
+description: Report a bug to help us improve
+title: "[BUG] <BUG>"
+labels: ["bug"]
+body:
+  - type: input
+    attributes:
+      label: Plugin
+      description: The affected NixVim plugin(s)
+  - type: dropdown
+    attributes:
+      label: Nixpkgs Release
+      multiple: false
+      options:
+        - "unstable"
+        - "25.05"
+        - "24.11"
+        - "other"
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Home Manager Release
+      multiple: false
+      options:
+        - "unstable"
+        - "25.05"
+        - "24.11"
+        - "other"
+  - type: checkboxes
+    attributes:
+      label: I have read the FAQ
+      options:
+        - label: >-
+            I have read the [FAQ](https://nix-community.github.io/nixvim/user-guide/faq.html)
+            and my bug is not listed there.
```

Does this support reference links?


```suggestion
            I have read the [FAQ] and my bug is not listed there.

            [FAQ]: https://nix-community.github.io/nixvim/user-guide/faq.html
```

---

it doesn't from my testing